developer-docs-site/docs/standards/fungible-asset.md

```diff
@@ -0,0 +1,451 @@
+---
+title: "Fungible Asset"
+id: "fungible-asset"
+---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Fungible Asset
+
+Fungible assets (FA) are an essential part of the Aptos ecosystem, as they enable the creation and transfer of fungible
+units, which can represent different things, such as currency, shares, material in games, or any other type of asset.
+Fungible assets can be used to build decentralized applications that require a token economy, such as decentralized
+exchanges or gaming platforms.
+
+[Fungible asset module](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/fungible_asset.move)
+provides a standard, typesafe framework for assets with fungibility.
+
+In this standard, fungible assets are stored in `Object<FungibleStore>` that has a specific amount of units, which
+can be transferred, burned, or minted. Fungible assets are units that are interchangeable with others of the same
+metadata. The standard is built upon object model so all the resources defined here are included in object resource
+group and stored inside objects.
+
+The relationship between the structures laid out in this standard is shown in this diagram.
+<div style={{textAlign: "center"}}>
+<ThemedImage
+alt="fungible asset architecture"
+sources={{
+    light: useBaseUrl('/img/docs/fungible-asset.svg'),
+    dark: useBaseUrl('/img/docs/fungible-asset-dark.svg'),
+  }}
+/>
+</div>
+
+## Difference with Aptos Coin
+
+FA is a broader category than just coins. While fungible coins are just one possible use case of FA, it can represent a wider range of fungible items, such as in-game assets, event tickets, and more. FA is constructed using an object model, which provides the flexibility for customizable, detailed management and offers a new programming model based on objects."
+The final goal of FA is to replace Aptos coin.
+
+## Structures
+
+### Metadata Object
+
+FA metadata defines the type of FA. A metadata is defined in Move as:
+
+```rust
+#[resource_group_member(group = aptos_framework::object::ObjectGroup)]
+struct Metadata has key {
+    supply: Option<Supply>,
+    /// Name of the fungible metadata, i.e., "USDT".
+    name: String,
+    /// Symbol of the fungible metadata, usually a shorter version of the name.
+    /// For example, Singapore Dollar is SGD.
+    symbol: String,
+    /// Number of decimals used for display purposes.
+    /// For example, if `decimals` equals `2`, a balance of `505` coins should
+    /// be displayed to a user as `5.05` (`505 / 10 ** 2`).
+    decimals: u8,
+}
+```
+
+:::tip
+This refers to a type system that uses metadata to differentiate between different types, much like CoinType in a coin standard. However, in this system, even if two metadata objects are exactly the same, they represent two distinct types rather than just one.
+:::
+
```

Comment:
```suggestion
```

Let's improve the language above, this tip implies we did not do a good job above. And I concur.